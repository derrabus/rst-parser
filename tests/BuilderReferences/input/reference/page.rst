Reference Page
==============

* `Some Sub Section`_
* :ref:`Other Sub Section <other>`
* :ref:`Section on another subpage <another-page>`

And what about :ref:`the test anchor <test_anchor>`?

Some Sub Section
----------------

.. _other:

Other Sub Section
-----------------
