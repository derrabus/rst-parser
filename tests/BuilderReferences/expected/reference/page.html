<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
    </head>
    <body>
        <div class="section" id="reference-page">
            <h1>Reference Page</h1>
            <ul>
                <li><a href="page.html#some-sub-section">Some Sub Section</a></li>
                <li><a href="page.html#other">Other Sub Section</a></li>
                <li><a href="page1.html#another-page">Section on another subpage</a></li>
            </ul>
            <p>And what about <a href="../reference.html#test_anchor">the test anchor</a>?</p>
            <div class="section" id="some-sub-section">
                <h2>Some Sub Section</h2>
            </div>
            <div class="section" id="other">
                <h2>Other Sub Section</h2>
            </div>
        </div>
    </body>
</html>